serverless/src/submitHarmonyOrder/handler.js

```diff
@@ -95,7 +95,6 @@ const submitHarmonyOrder = async (event, context) => {
         environment
       })
 
-      console.log('Harmony order payload')
```

Comment:
This looks like a log that shouldn't have been removed

Reply:
I was going to ask about that do you think we should actually include the object otherwise its just a string. Here's an example of what it resolves to `🚀 ~ file: handler.js:100 ~ awaitsqsRecords.forEachAsync ~ orderPayload: FormData
forceAsync: true
granuleId: G1261505122-LPCLOUDUAT,G1261505131-LPCLOUDUAT,G1261505133-LPCLOUDUAT,G1261505140-LPCLOUDUAT,G1261505124-LPCLOUDUAT,G1261505130-LPCLOUDUAT,G1261505132-LPCLOUDUAT,G1261505138-LPCLOUDUAT,G1261505125-LPCLOUDUAT,G1261505128-LPCLOUDUAT,G1261505134-LPCLOUDUAT,G1261505139-LPCLOUDUAT,G1261505127-LPCLOUDUAT,G1261505129-LPCLOUDUAT,G1261505136-LPCLOUDUAT,G1261505137-LPCLOUDUAT,G1261505103-LPCLOUDUAT,G1261505123-LPCLOUDUAT,G1261505126-LPCLOUDUAT,G1261505135-LPCLOUDUAT`

Reply:
Actually this does tell us how far along the code is in I'm putting it back in